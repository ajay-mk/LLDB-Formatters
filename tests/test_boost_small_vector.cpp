#include <boost/container/small_vector.hpp>
#include <iostream>

int main() {
    boost::container::small_vector<int, 4> vec;
    vec.push_back(10);
    vec.push_back(20);
    vec.push_back(30);
    std::cout << "pause" << std::endl; // For breakpoint
    return 0;
}