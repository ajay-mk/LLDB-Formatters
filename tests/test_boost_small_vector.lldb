command script import ./boost_formatter.py
break set --name main
run
frame variable vec
quit