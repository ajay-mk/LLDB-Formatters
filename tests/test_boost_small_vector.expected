(lldb) frame variable vec
(boost::container::small_vector<int, 4>) vec = {
  [0] = 10
  [1] = 20
  [2] = 30
}